lang/python/python-six/Makefile

```diff
@@ -66,6 +66,7 @@ endef
 
 define Host/Compile
 	$(call Build/Compile/HostPyMod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
+	$(call Build/Compile/HostPy3Mod,,install --prefix="" --root="$(STAGING_DIR_HOSTPKG)")
```

Comment:
you can switch to the `Build/Compile/HostPy3Mod` & `python3/host` completely here;
no need to add both;

OVS is the only user of `python-six/host`; this is a left-over from when I was maintaining OVS ;
I never made time to migrate it to a newer format

Reply:
Thanks for the info.  I will check again and update